lib/ansible/modules/packaging/os/yum.py

```diff
@@ -988,10 +1011,13 @@ def latest(module, items, repoq, yum_basecmd, conf_file, en_repos, dis_repos, in
         for w in will_update:
             if w.startswith('@'):
                 to_update.append((w, None))
-                msg = '%s will be updated' % w
+                res['msg'] += '%s will be updated' % w
```

Comment:
not even sure why that previous msg = '%s will be updated' % w was there, just tried to 'guess'